integration/cri-o/crio_skip_tests.sh

```diff
@@ -27,6 +31,94 @@ declare -A skipCRIOTests=(
 ['test "annotations passed through"']='This is not working'
 ['test "ctr with default_env set in configuration"']='This is not working'
 ['test "ctr with absent mount that should be rejected"']='This is not working'
+['test "pids limit"']='cgroups.bats case is not working'
```

Comment:
Please add a GitHub issue URL for each test so the reader can find out _why_ each test is not working.

Reply:
This is where I wasn't sure : we are listing the failures, but we did not triage them yet.
Some may be false positives, some may be kata issues, and some may be cri-o issues.
At the moment, Xiangchun is listing them in our bug tracker (Jira) for triaging, and we plan to review and log them in the appropriate place where needed, but it is going to take more time.
This commit allows us to add passing tests to our CI, and increase its coverage before we start fixing the failures.

So - do you think we should log all issues in GitHub before we triage them (with the risk of flooding with non-kata issues) ? 
Or maybe link to our Jira (which is public anyway) until triage is done ?
Any suggestion is welcome, I just want to avoid making too much noise.

Reply:
In my personal opinion, I think add one github issue per skipped test and update in the script would be overkill (but hey @xiangchunfu, if you want to do that I would be fine with it! ;-) ) 
What about opening one single github issue for tracking all the skipped tests, and reference it in a comment here?
In the single "main" github issue optionally we can list the Jira issues for now: but when one will triage one or more of the skipped test, the  "main" github issue can be updated (or a new one may be open and linked to it).

Reply:
> In my personal opinion, I think add one github issue per skipped test and update in the script would be overkill (but hey @xiangchunfu, if you want to do that I would be fine with it! ;-) ) What about opening one single github issue for tracking all the skipped tests, and reference it in a comment here? In the single "main" github issue optionally we can list the Jira issues for now: but when one will triage one or more of the skipped test, the "main" github issue can be updated (or a new one may be open and linked to it).

I try to create a github jira issue to track all skipped test cases as blew.  Is it expected?
https://github.com/kata-containers/tests/issues/4459

Reply:
A single issue is fine with me, but could you update the messages to something like:

```suggestion
url=https://github.com/kata-containers/tests/issues/4459
['test "pids limit"']='cgroups.bats case is not working: see $url'
```

Reply:
@jamesodhunt Thanks, I add the message with comments in 53~54 line.